pythonforandroid/bootstrap.py

```diff
@@ -263,11 +263,11 @@ def strip_libraries(self, arch):
             info('Python was loaded from CrystaX, skipping strip')
             return
         env = arch.get_env()
-        strip = which('arm-linux-androideabi-strip', env['PATH'])
+        strip = env['STRIP'].split(' ')[0]
```

Comment:
I don't think this `split` is a good idea. Also see my comment in the conversation

Reply:
> it's not unthinkable the strip command contains other options that are needed. (e.g. I'm pretty sure it's not completely unthinkable a more unusual compiler toolchain might require something like `compiler-name --strip` to invoke a strip tool) So that seems like a bad idea to me.

This is a very important point that I wasn't well aware of.  In 73d13ad, I have updated code so that the `strip` command object reflects the original `STRIP` command specified in `env['STRIP']` including option strings.  This way, we don't need to add `--strip-unneeded` in this part again (already done in archs.py).

Reply:
I like the functionality you have now, but I would suggest writing it a little simpler:

```
tokens = env['STRIP'].split(' ')
strip = sh.Command(tokens[0]).bake(tokens[1:])
```
(also works when `len(tokens) == 1`/no options are used)

Reply:
Wait, I'm just noticing that will break with space character arguments. I think this should be done instead:

```
tokens = shlex.split(env['STRIP'])
strip = sh.Command(tokens[0]).bake(tokens[1:])
```